crates/voicevox_core_node_api/Cargo.toml

```
@@ -0,0 +1,23 @@
+[package]
+name = "voicevox_core_node_api"
+version = "0.0.0"
+edition.workspace = true
+publish.workspace = true
+
+[lib]
+crate-type = ["cdylib"]
+
+[features]
+cuda = ["voicevox_core/cuda"]
+directml = ["voicevox_core/directml"]
+
+[dependencies]
+napi = { version = "2.16.0", features = ["napi4", "serde-json", "tokio_rt"] }
+napi-derive = "2.16.0"
+voicevox_core.workspace = true
```

<https://github.com/VOICEVOX/voicevox_core/pull/802#issuecomment-2209653063>

> - [featureとして`load-onnxruntime`と`link-onnxruntime`](https://voicevox.github.io/voicevox_core/apis/rust_api/voicevox_core/index.html#onnx-runtime%E3%81%AE%E3%83%AA%E3%83%B3%E3%82%AF%E6%96%B9%E6%B3%95%E3%82%92%E6%B1%BA%E3%82%81%E3%82%8B%E3%83%95%E3%82%A3%E3%83%BC%E3%83%81%E3%83%A3)が追加されました。ただしPythonとJavaでは`load-onnxruntime`で固定です。Node.jsもそうすることになると思います。

```suggestion
voicevox_core = { workspace = true, features = ["load-onnxruntime"] }
```